stable/elasticsearch-exporter/templates/deployment.yaml

```diff
@@ -89,9 +89,11 @@ spec:
                     "--es.ssl-skip-verify",
                     {{- end }}
                     {{- if .Values.es.ssl.enabled }}
-                    "--es.ca={{.Values.es.ssl.ca.path | default "/ssl/ca.pem" }}",
-                    "--es.client-cert={{.Values.es.ssl.client.pemPath | default "/ssl/client.pem" }}",
-                    "--es.client-private-key={{.Values.es.ssl.client.keyPath | default "/ssl/client.key" }}",
+                    "--es.ca={{.Values.es.ssl.ca.path }}",
+                    {{- if and .Values.es.ssl.client.pemPath .Values.es.ssl.client.keyPath }}
```

Comment:
Maybe it would be would be more straightforward to have a boolean flag like `{{- if .Values.es.ssl.client.enabled }}` (with default set to "true") which is also properly document in the chart arguments listed in the README file? Looking at the current README or values.yaml it's not clear how to disable the client SSL cert (only by looking at the used template).

Reply:
... haven't thought of that :|

Thanks @svenmueller, code updated as suggested.